Gemfile

```diff
@@ -2,3 +2,12 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in amazon_ssa_support.gemspec
 gemspec
+
+gem "aws-sdk"
```

Comment:
@hsong-rh I'm not saying just *remove* them - I'm saying move them from the Gemfile to the gemspec as we do for the other plugins.  Bundle install should work the same either way.  Unless I am misinterpreting what is happening.